lucene/facet/src/java/org/apache/lucene/facet/taxonomy/directory/DirectoryTaxonomyReader.java

```diff
@@ -314,7 +316,6 @@ public int getOrdinal(FacetLabel cp) throws IOException {
 
   @Override
   public FacetLabel getPath(int ordinal) throws IOException {
-    ensureOpen();
```

Comment:
Hmm why is this removed?

Reply:
In the earlier iteration it was shifted to the helper method that looked up values in the cache. I have kept it in the outer method in the most recent commit